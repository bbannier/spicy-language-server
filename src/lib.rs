#![feature(str_strip)]

#[macro_use]
extern crate pest;

pub mod lsp;
pub mod options;
pub mod spicy;
