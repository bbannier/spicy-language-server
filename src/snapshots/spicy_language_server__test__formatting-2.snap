---
source: src/lib.rs
expression: "server.formatting(DocumentFormattingParams\n{\n    text_document:\n    TextDocumentIdentifier::new(Url::from_file_path(\"/does_not_exist.spicy\").unwrap()),\n    options: FormattingOptions::default(), work_done_progress_params:\n    WorkDoneProgressParams::default(),\n}).await"
---
Ok(
    None,
)
